src/components/WhimsicalInstaller/WhimsicalInstaller.js

```diff
@@ -95,6 +88,34 @@ class WhimsicalInstaller extends PureComponent<Props, State> {
     y: this.getHeight() * 0.5 - 4,
   });
 
+  createFile = () => {
+    /**
+     * Create a single new file, from the planet's center
+     */
+    const { width } = this.props;
+    const height = this.getHeight();
+
+    const fileId = uuid();
+
+    const startingPoint = this.getPlanetPoint();
+
+    this.setState(state => {
+      return {
+        files: {
+          ...state.files,
+          [fileId]: {
+            id: fileId,
+            x: startingPoint.x,
+            y: startingPoint.y,
+            status: 'autonomous',
+            size: height / 4,
+            flightPath: generateFlightPath(width, height),
+          },
+        },
+      };
+    });
```

Comment:
This isn't a new method, just moved from below, + now it actually sets the state to match the update/delete methods